features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -370,19 +373,31 @@ int lwip_bringup(bool dhcp, const char *ip, const char *netmask, const char *gw)
         sys_arch_sem_wait(&lwip_tcpip_inited, 0);
 
         memset(&lwip_netif, 0, sizeof lwip_netif);
+#if DEVICE_EMAC
```

Comment:
Can you share the code snippet with a preload ? With the former suggestion , it would be:

```
`
        if (!netif_add(&lwip_netif,
#if LWIP_IPV4
                0, 0, 0,
#endif
#if DEVICE_EMAC
                emac, emac_lwip_if_init,
#else
                NULL, eth_arch_enetif_init,
#endif
                tcpip_input)) {
            return -1;
        }

```